abs.yaml

```diff
@@ -0,0 +1,4 @@
+upload_channels:
+  - sfe1ed40
+channels:
+  - sfe1ed40
```

Comment:
```suggestion
```

Reply:
I need this to pull `wirerope` from the channel

Reply:
using only `upload_channels ` would be enough to source dependencies from the same channel